nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -135,29 +89,17 @@
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_MAXIMUM_MSG_SIZE = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_MSG_SIZE = new PropertyDescriptor.Builder()
```

Comment:
Is SMTP_MAXIMUM_SIZE being enforced? I've set it to 512 B and ended up with a message of 1KB?

Note that setMaxMessageSize has no effective role in dropping messages, instead all it does is to configure the EHLO response:

```
$ telnet 0 2525
Trying 0.0.0.0...
Connected to 0.
Escape character is '^]'.
220 localhost ESMTP Apache NiFi
EHLO .
250-localhost
250-8BITMIME
250-SIZE 512
250 Ok

```